website/docs/operations/02.1-timers.mdx

```diff
@@ -12,3 +12,16 @@ This operation enables time-based coordination, allowing workflows to pause for
 ```
 
 <OperationOutputs name="timer"/>
+## Drafting Support
+
+When drafting workflows that need time-based operations, use `WIO.draft.timer()`:
+
+```scala file=./main/scala/workflows4s/example/docs/DraftTimerExample.scala start=start_draft end=end_draft
+```
+
+<OperationOutputs name="draft-timer"/>
+
+The draft timer operation simplifies the API by:
+- Not requiring explicit event handling
+- Making duration parameter optional (can be determined at runtime)
```

Comment:
```suggestion
- Making duration parameter optional
```

I would drop the explanation, because there is no runtime doir drafts (they are not runnable).